apps/src/templates/BaseDialog.jsx

```diff
@@ -54,6 +55,12 @@ var BaseDialog = React.createClass({
         marginLeft: 0,
       };
     }
+    if (this.props.fullWidth) {
+      bodyStyle = Object.assign({}, bodyStyle, {
+        width: '90%',
+        marginLeft: '-45%'
+      });
+    }
```

Comment:
I tried that, and for cases where `bodyStyle` is undefined it doesn't work, where this does.